charts/harmony-chart/values.yaml

```diff
@@ -235,3 +237,70 @@ karpenter:
   # Node template reference. More details in https://karpenter.sh/docs/concepts/node-templates/
   nodeTemplate:
     name: "default"
+
+# Prometheus stack
+prometheusstack:
```

Comment:
If no storage options are specified, Prometheus and Grafana data are stored in emptyDir (ephemeral). Should we add, maybe in a comment, the instructions to add PVC's to Prometheus and Grafana?

Reply:
Thank you, added a comment!